core/sis-utility/src/main/java/org/apache/sis/measure/Scalar.java

```diff
@@ -539,4 +540,17 @@ public final String toString() {
             return new Temperature(value, unit);
         }
     }
+
+    static final class Acceleration extends Scalar<javax.measure.quantity.Acceleration>
```

Comment:
This definition can be moved just below `Speed` for keeping related quantities together.